rct229/rulesets/ashrae9012019/ruleset_functions/get_hvac_systems_primarily_serving_comp_room_test.py

```diff
@@ -0,0 +1,96 @@
+                                {
+                                    "id": "space_1_1_1",
+                                    "floor_area": 1000,  # m2
+                                    "lighting_space_type": "COMPUTER_ROOM",
+                                    "occupant_multiplier_schedule": "om_sched_1",
+                                },
+                                {
+                                    "id": "space_1_1_2",
+                                    "floor_area": 500,  # m2
+                                    "occupant_multiplier_schedule": "om_sched_1",
+                                },
+                            ],
+                            "terminals": [
+                                {
+                                    "id": "terminal_1_1_1",
+                                    "served_by_heating_ventilating_air_conditioning_system": "hvac_1",
+                                },
+                                {
+                                    "id": "terminal_1_1_2",
+                                    "served_by_heating_ventilating_air_conditioning_system": "hvac_1",
+                                },
+                            ],
+                        },
+                        {
+                            "id": "zone_1_2",
+                            "thermostat_cooling_setpoint_schedule": "tcs_sched_1",
+                            "thermostat_heating_setpoint_schedule": "ths_sched_1",
+                            "spaces": [
+                                {
+                                    "id": "space_2_1_1",
+                                    "floor_area": 1000,  # m2
+                                    "occupant_multiplier_schedule": "om_sched_1",
+                                },
+                                {
+                                    "id": "space_2_1_2",
+                                    "floor_area": 500,  # m2
+                                    "occupant_multiplier_schedule": "om_sched_1",
+                                },
+                            ],
+                            "terminals": [
+                                {
+                                    "id": "terminal_2_1_1",
+                                    "served_by_heating_ventilating_air_conditioning_system": "hvac_2",
+                                },
+                                {
+                                    "id": "terminal_2_1_2",
+                                    "served_by_heating_ventilating_air_conditioning_system": "hvac_2",
+                                },
+                            ],
+                        },
+                    ],
+                }
+            ],
+        }
+    ],
+}
+TEST_RMD = {"id": "229_01", "ruleset_model_instances": [TEST_RMI]}
+
+TEST_RMI_UNIT = quantify_rmr(TEST_RMD)["ruleset_model_instances"][0]
+
+
+def test__TEST_RMD__is_valid():
+    schema_validation_result = schema_validate_rmr(TEST_RMI_UNIT)
```

Comment:
To get the pytest passed, I believe ````TEST_RMI_UNIT``` should be replaced with ```TEST_RMD```. Could you check this out?